conan_v2.rst

```diff
@@ -71,9 +71,14 @@ available for Conan 2.0.
 Host and build profiles and new cross-building model
 ----------------------------------------------------
 
-Use always build and host profiles. You can enable it by passing ``-pr:b=default`` in the
-command line to most commands. Do not use ``os_build``, ``arch_build`` anywhere in your
-recipes or code.
+Use always build and host profiles.
```

Comment:
maybe we can add a link to: https://docs.conan.io/en/latest/reference/profiles.html#build-profiles-and-host-profiles ?

Reply:
sure!

Reply:
oh, you meant in that paragraph, wait 1 min